config/environments/development.rb

```diff
@@ -72,4 +72,7 @@
       :authentication => :plain,
       :enable_starttls_auto => false
   }
+
+  # Allow console to connect from docker
+  config.web_console.whitelisted_ips = ['172.29.0.0/16', '127.0.0.1']
```

Comment:
Why we need to have `172.29.0.0/16`?

Reply:
I believe this is for the docker IP so that the web-console can run when an error is encountered on the webpages error page